vnext/Desktop/Modules/BlobModule.cpp

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "BlobModule.h"
+
+#include <unicode.h>
+
+// React Native
+#include <cxxreact/Instance.h>
+#include <cxxreact/JsArgumentHelpers.h>
+#include <cxxreact/ModuleRegistry.h>
+
+// Windows API
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Security.Cryptography.h>
+
+using namespace facebook::xplat;
+
+using folly::dynamic;
+using std::lock_guard;
+using std::mutex;
```

Comment:
I know this is a stylistic thing, but from the last discussion we were trying to align anything possible to CppCoreGuidelines, which discourages `using` for short namespaces. I.e. there isn't any other code in the repo written by other folks who bring in `std`. Would appreciate if we could keep consistent styling, since seeing raw `string`, etc can trip people up.

Reply:
We should follow up offline.
I was not aware we had an official guideline source.

I thought we were using a slightly modified facebook formatting convention.
Being able to strip namespaces allows fewer word-wraps forced by clang format.

Reply:
It also lends itself to symbol collisions. Take for example folly - we have `using namespace folly` everywhere. Folly added a new type called `tag`, and we use the variable name `tag` in lots of places. Now the compiler is confused about what we mean by `tag`. It also makes it hard to reason about all usages of a symbol across the codebase.

Reply:
> It also lends itself to symbol collisions. Take for example folly - we have `using namespace folly` everywhere. Folly added a new type called `tag`, and we use the variable name `tag` in lots of places. Now the compiler is confused about what we mean by `tag`. It also makes it hard to reason about all usages of a symbol across the codebase.

This case is different. I'm not including the whole `folly` namespace, but only the `dynamic` type.
Collisions with other folly types or names shouldn't happen at all.

Reply:
It does still increase the chance of collision in cases where ADL is used (see https://en.cppreference.com/w/cpp/language/adl). That's admittedly super nitpicky.

Ive personally found short namespaces to be useful locally to disambiguate when reading code, since we have a lot of the same identifiers in winrt, STL, boost, etc. It's definitely a little more wordy, but imo better than the alternative, and is more consistent with the rest of the codebase, and most other C++ codebases I've worked in.

Usually if Clang Format breaks earlier than I'd like, I can look to see if I'm doing too much on one line, or are using names that might be too verbose. The 120 char line limit we have is liberal enough imo. Some of it is also just stylistic where you kind of get used to how it breaks things.

Reply:
Ultimately, any namespace usage (long or short) increases the risk of collisions.
The core of the matter is "how much".

For names like `dynamic`, `string`, `mutex` or `lock_guard`, I'd argue not very. Especially if limited to this compilation unit.

Reply:
When in doubt, it's usually best to conform to any existing precedent set. A lot of this things are inherently subjective, but it's almost always better to be a team player to create a consistent codebase.